### PR TITLE
[bitnami/*] Fixing the inconsistencies

### PR DESCRIPTION
**Pull request by superaleks**
Signed-off-by: alukic <alukic@vmware.com>

**Description of the change**

Creating consistency across the files and folders used for Helm Chart testing strategy. Applying naming convention decided by the team.

**Description of the change**

Tests:

Please note that the test results express the validity of the workflow and test execution, not the test results themselves. 

1. PHPMyAdmin -  [PASS](https://github.com/superaleks/charts/runs/5985923481?check_suite_focus=true)
2. Airflow - [PASS](https://github.com/superaleks/charts/runs/5986516938?check_suite_focus=true)
3. Grafana - [PASS](https://github.com/superaleks/charts/runs/5986875638?check_suite_focus=true)
4. Harbor - [PASS](https://github.com/superaleks/charts/runs/5987276039?check_suite_focus=true)
5. Kibana - [PASS](https://github.com/superaleks/charts/runs/5988084991?check_suite_focus=true)
6. MiniO - [PASS](https://github.com/superaleks/charts/runs/5991482276?check_suite_focus=true)
7. Redis - [PASS](https://github.com/superaleks/charts/pull/60/checks)
8. SolR - [PASS](https://github.com/superaleks/charts/runs/5988833467?check_suite_focus=true)
9. Thanos - [PASS](https://github.com/superaleks/charts/pull/63/checks)
10. WordPress - [PASS](https://github.com/superaleks/charts/runs/5990249373?check_suite_focus=true)

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .github/workflows/vib-airflow.yaml [hunk 1]
```diff
@@ -10,27 +10,23 @@ env:
   CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
   VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
 jobs:
-  vib-k8s-verify: # verify in multiple target platforms
+  vib-k8s-verify:
```
[the remark this answers is not in the document]
Comment (superaleks) [reply .github/workflows/vib-airflow.yaml]: Good catch, I've applied the changes.

**Review by recena — DISMISSED** [review]

File: .github/workflows/vib-minio.yaml [hunk 2]
```diff
@@ -10,27 +10,23 @@ env:
...
            target-platform-id: 7ddab896-2e4e-4d58-a501-f79897eba3a0
       fail-fast: false
-    name: K8s verify ${{ matrix.target-platform}}
+    name: MiniO on ${{ matrix.target-platform }}
     steps:
       - uses: actions/checkout@v2
         with:
           ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
+          repository: ${{github.event.pull_request.head.repo.full_name }}
```
Comment (FraPazGal) [inline .github/workflows/vib-minio.yaml:27]:
```suggestion
          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

File: .github/workflows/vib-phpmyadmin.yaml [hunk 3]
```diff
@@ -10,27 +10,23 @@ env:
...
       fail-fast: false
-    name: K8s verify ${{ matrix.target-platform}}
+    name: PhpMyAdmin on ${{ matrix.target-platform }}
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name }}
```
Comment (FraPazGal) [inline .github/workflows/vib-phpmyadmin.yaml:27]:
```suggestion
          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

File: .github/workflows/vib-solr.yaml [hunk 4]
```diff
@@ -10,27 +10,23 @@ env:
...
       fail-fast: false
-    name: K8s verify ${{ matrix.target-platform}}
+    name: SolR on ${{ matrix.target-platform }}
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name }}
```
Comment (FraPazGal) [inline .github/workflows/vib-solr.yaml:27]:
```suggestion
          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

**Review by FraPazGal — COMMENTED** [review]
A related comment, we may want to have our GH's vib-related workflows inside a folder `charts/.github/vib/` to keep them more organized and avoid having ~90 `vib-*.yaml` in the same directory where we have the older (but still used) GH workflows.

File: .github/workflows/vib-thanos.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    branches:
+      - master
+    paths:
+      - 'bitnami/thanos/*'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  vib-bucketweb:
```
Comment (joancafom) [inline .github/workflows/vib-thanos.yaml:13]:
```suggestion
  vib-verify-bucketweb:
```
In order to follow the same convention for multiple jobs, as in `vib-redis.yaml`

File: .github/workflows/vib-thanos.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - uses: vmware-labs/vmware-image-builder-action@main
+        with:
+          pipeline: thanos/bucketweb/vib-complete-verify.json
+        env:
+          VIB_ENV_TARGET_PLATFORM: 7ddab896-2e4e-4d58-a501-f79897eba3a0
+
+  vib-queryfrontend:
```
Comment (joancafom) [inline .github/workflows/vib-thanos.yaml:28]:
```suggestion
  vib-verify-queryfrontend: 
```

Same comment here

File: .github/workflows/vib-thanos.yaml (hunk 5 above)
Comment (superaleks) [reply .github/workflows/vib-thanos.yaml]: Great catch! Fixed it!

**Review by joancafom — APPROVED** [review]
LGTM

**Pull request merged**